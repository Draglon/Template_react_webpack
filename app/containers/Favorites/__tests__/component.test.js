import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import FavoritesComponent from '../component';

describe('<FavoritesComponent /> container', () => {
  let component;

  it('render property', () => {
    component = shallow(<FavoritesComponent />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
