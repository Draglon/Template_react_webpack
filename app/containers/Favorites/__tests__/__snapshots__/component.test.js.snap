// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeComponent /> container render property - with search query 1`] = `
<Fragment>
  <SearchComponent
    onSearch={[MockFunction]}
  />
  <MovieListComponent
    movies={Array []}
  />
  <PaginationComponent
    currentPage={1}
    getPage={[MockFunction]}
    query="some string"
    totalPages={20}
  />
</Fragment>
`;

exports[`<HomeComponent /> container render property - without search query 1`] = `
<Fragment>
  <SearchComponent
    onSearch={[MockFunction]}
  />
  <MovieListComponent
    movies={Array []}
  />
  <PaginationComponent
    currentPage={1}
    getPage={
      [MockFunction] {
        "calls": Array [
          Array [
            Object {
              "page": 1,
            },
          ],
        ],
        "results": Array [
          Object {
            "type": "return",
            "value": undefined,
          },
        ],
      }
    }
    query=""
    totalPages={100}
  />
</Fragment>
`;
